test/harness/e2e/harness.go

```diff
@@ -107,3 +130,37 @@ func (h *Harness) ApproveEnrollment(id string, approval *v1alpha1.EnrollmentRequ
 	Expect(apr.JSON200).NotTo(BeNil())
 	logrus.Infof("Approved device enrollment: %s", id)
 }
+
+func (h *Harness) ApiEndpoint() string {
```

Comment:
Right, I should stick to the same thing everywhere (i.e. logrus like the parts above (see L131) )instead of printf.

Reply:
I will check Setenv for error, although that is extremely unlikely to happen, we should handle errors, I agree.

If an error happens getting the external IP it will fail here: https://github.com/flightctl/flightctl/pull/410/files#diff-796b21d55382ce34f5acd0e5edb354989cebcf8591aa04396b461e0092bbd338R37